### DISCUSSION TITLE
morphia-core-2.3.0-SNAPSHOT.jar on WildFly 26.x

### DISCUSSION THREAD

**Discussion by NOTiFYcouk** [MorphiaOrg/morphia#1874] [General]
Does  "Morphia" need to be deployed on XML with "morphia-core-2.3.0-SNAPSHOT.jar" on WildFly (26.x) as on:

`module / system / layers /  base / dev / morphia / main / `

Where I'd put:

```
morphia-core-2.3.0-SNAPSHOT.jar  &
module.xml
```
Deploying "morphia' as an EJB.

**Comment by evanchooly** [comment]
I don't honestly. I know folks on the widlfy team so I can ask around but can't promise anything.

Any reason to don't just include that jar in your war/ear?

**Comment by NOTiFYcouk** [comment]
You're right. It will deployed on EAR:

```
dependencies {
    earlib group: "dev.morphia.morphia", name: "morphia-core", version: "2.3.0-SNAPSHOT", ext: "jar"
}
```
